### PR TITLE
Feature evaluation bechmarks

### PR DESCRIPTION
**Pull request by jano1906**

### CODE REVIEW

File: experiments/data/extract_features.py [hunk 1]
```diff
@@ -49,18 +75,23 @@ def extract_features(model: SmilesEncoder, data: list[str], output_path: str, ch
     for f in files:
         os.remove(f)
 
+
 def main(args):
     logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
     model_config = ModelConfig.from_config_file(args.path_to_model_config)
     task_config = TaskConfig.from_config_file(args.path_to_task_config)
```
Comment (AdamIzdebski) [inline experiments/data/extract_features.py:82]: change to DatasetConfig and TokenizerConfig

File: experiments/joint_training/features_eval.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        super().__init__()
+        self.data = torch.from_numpy(np.load(data_path))
+        self.target = torch.from_numpy(np.load(target_path))
+        assert len(self.data) == len(self.target)
+
+    def __len__(self):
+        return len(self.data)
+    
+    def __getitem__(self, index):
+        return self.data[index], self.target[index]
```
Comment (AdamIzdebski) [inline experiments/joint_training/features_eval.py:29]: #34 for later

File: jointformer/models/moler.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import numpy as np
 from molecule_generation.utils.model_utils import load_vae_model_and_dataset
 from molecule_generation.utils.moler_inference_server import _encode_from_smiles
-
+import tensorflow as tf
```
Comment (AdamIzdebski) [inline jointformer/models/moler.py:11]: will this require installing tf even without using Moler? No, right?

File: experiments/joint_training/features_eval.py (hunk 2 above)
Comment (jano1906) [reply experiments/joint_training/features_eval.py]: fcb5942ca49a89f60ff99be83e9f2f75aac71d1a

File: jointformer/models/moler.py (hunk 3 above)
Comment (jano1906) [reply jointformer/models/moler.py:11]: No, unless jointformer.models.moler is imported

**Review by AdamIzdebski — APPROVED** [review]

**Pull request merged**